core/src/main/java/com/yahoo/ycsb/generator/LongGenerator.java

```diff
@@ -0,0 +1,77 @@
+/**                                                                                                                                                                                
+ * Copyright (c) 2013 Yahoo! Inc. All rights reserved.
+ *                                                                                                                                                                                 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you                                                                                                             
+ * may not use this file except in compliance with the License. You                                                                                                                
+ * may obtain a copy of the License at                                                                                                                                             
+ *                                                                                                                                                                                 
+ * http://www.apache.org/licenses/LICENSE-2.0                                                                                                                                      
+ *                                                                                                                                                                                 
+ * Unless required by applicable law or agreed to in writing, software                                                                                                             
+ * distributed under the License is distributed on an "AS IS" BASIS,                                                                                                               
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or                                                                                                                 
+ * implied. See the License for the specific language governing                                                                                                                    
+ * permissions and limitations under the License. See accompanying                                                                                                                 
+ * LICENSE file.                                                                                                                                                                   
+ */
+
+package com.yahoo.ycsb.generator;
+
+/**
+ * A generator that is capable of generating ints as well as strings
+ * 
+ * @author cooperb
+ *
+ */
+public abstract class LongGenerator extends Generator
```

Comment:
To maintain compatibility, you could have LongGenerator extend IntegerGenerator so that all the current implementers will stay have IntegerGenerator as an ancestor (and having getInt will make sense)